client/Scenarios/ShutdownVmsByTag.cs

```diff
@@ -18,7 +18,7 @@ public override void Execute()
             Random rand = new Random(Environment.TickCount);
             foreach (var generic in rg.VirtualMachines().ListByName(Environment.UserName))
             {
-                var vm = new VirtualMachineOperations(generic);
+                var vm = VirtualMachineOperations.FromGeneric(generic);
```

Comment:
We decided on the FromGeneric pattern then?

Reply:
No, this is the straw man that allows us to maintain all constructors internal for now.  This is still consideration